### DISCUSSION TITLE
Moving action buttons below a image (?) / Переместить кнопки под изображение (?)

### DISCUSSION THREAD

**Discussion by paul-k-pastvu** [PastVu/pastvu#168] [Q&A / Вопросы и ответы]
See comment: https://pastvu.com/news/129?hl=comment-957725

EN: “But the image editing buttons would also be placed under the photo. Why ... Because after editing, when the page is pushed to them on tablets, the page starts updating and everything that has been edited disappears. This is a purely convenient property of browsers implemented in tablets, but it contradicts the layout of the pages of this site. "

RU: "А вот кнопки редактирования фото поместить бы тоже под фотографией. Почему...
Потому что после редактирования при продергивании страницы до них на планшетах запускается обновление страницы и всё, что отредактировал, пропадает.
Это чисто удобное свойство браузеров, реализованное в планшетах, но оно противоречит устройству страниц этого сайта."

**Comment by paul-k-pastvu** [comment]
Мне кажется, что сделано несколько в лоб. Нужно обсуждать такие вещи.
Это не зря было записано в задачи с тегом "Question".
То, что удобно пользователям планшетов, неудобно пользователям ПК. Как минимум, это меняет привычные сценарии использования. Такое нужно анонсировать заранее, и внимательно выслушивать мнения.

**Comment by aeifn** [comment]
Reverted: 
05e053a9a1604a82419bcc0799961505a70e6347
https://github.com/PastVu/pastvu/commit/ed6e19fba62ac5429f738e346bc09071ce4ab085
